cime_config/cesm/machines/config_machines.xml

```diff
@@ -218,18 +218,18 @@
     <mpirun mpilib="mvapich2">
       <executable>srun</executable>
       <arguments>
-        <arg name="mpi">--mpi=none</arg>
```

Comment:
Just wanted to repeat the comment that the indenting changes in this file are confusing to me.